### DISCUSSION TITLE
How do you access the host file system

### DISCUSSION THREAD

**Discussion by Carl-P4** [Jip-Hop/ubernerd#5] [Q&A] [answered]
With Truenas Scale, I want to run portainer to control docker containers that reference locations on the host file system.
Currently I have a script I have to run after a reset to copy my docker/daemon.json and restart docker, I have portainer running with my containers.

Want this to work on boot, so l found your repo,
Is this possible with your guide to install a Persistent container that you can then install docker inside of and for the docker containers to have access to the host file system?

**Comment by Jip-Hop** [comment, marked as answer]
It can work as you describe. Although I would recommend [jailmaker](https://github.com/Jip-Hop/jailmaker) at the moment for this purpose. It's easier to install, maintain, has more users and is safer to run alongside TrueNAS Apps. I use jailmaker with a single persistent debian container in which I installed docker and portainer. Has been working great for months.

**Comment by florihupf** [comment]
I am just here to second that statement. On Oct 9, 2023, at 08:42, Jip-Hop ***@***.***> wrote:﻿
It can work as you describe. Although I would recommend jailmaker at the moment for this purpose. It's easier to install, maintain, has more users and is safer to run alongside TrueNAS Apps. I use jailmaker with a single persistent debian container in which I installed docker and portainer. Has been working great for months.
